src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,142 @@
+package duke;
+
+import duke.task.Task;
+import org.json.JSONArray;
```

Comment:
I think it is cool that you have used JSON objects to store your data and it is definitely something I can learn from!